SCloudRandomizer/MainViewController.m

```diff
@@ -132,7 +114,30 @@ - (IBAction)logout:(id)sender {
     [self.lblTitleValue setHidden:YES];
     [self.lblArtistValue setHidden:YES];
     [self.lblLengthValue setHidden:YES];
+}
+
+- (void) hidePlayerControls {
```

Comment:
Dont need this function. Just put this stuff in hideAllDataAndControls. Too much encapsulation.